apps/dash-image-annotation/dash-bounding-box-image-annotation/README.md

```diff
@@ -0,0 +1,38 @@
+# Dash Bounding Box Image Annotation
+
+## About this app
+
+This shows how Dash can be used to make an application that collects annotations
+of the content in parts of images. These annotations could be downloaded and
+used to train an image content classifier.
+
+## How to run this app locally
+
+(The following instructions are for unix-like systems)
```

Comment:
why? On Windows Anaconda for example provides a unix-like shell which can be used to execute the instructions below.